paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -379,24 +385,34 @@ internal class PaymentSheetActivityTest {
             // wait for bottom sheet to animate in
             idleLooper()
 
-            // New valid card
-            val initialSelection = PaymentSelection.New.Card(
-                PaymentMethodCreateParamsFixtures.DEFAULT_CARD,
-                CardBrand.Visa,
-                customerRequestedSave = PaymentSelection.CustomerRequestedSave.RequestNoReuse
-            )
-            viewModel.updateSelection(initialSelection)
-
-            viewModel.transitionToAddPaymentScreen()
+            runTest {
+                val initialSelection = PaymentSelection.New.Card(
+                    PaymentMethodCreateParamsFixtures.DEFAULT_CARD,
+                    CardBrand.Visa,
+                    customerRequestedSave = PaymentSelection.CustomerRequestedSave.RequestNoReuse
+                )
```

Comment:
This test is confusing to me (and already before your change). Since we seem to only test view model things, can we move this into the view model test?